client/post-editor/editor-ground-control/index.jsx

```diff
@@ -91,10 +93,23 @@ export default React.createClass( {
 			showDateTooltip: false,
 			firstDayOfTheMonth: this.getFirstDayOfTheMonth(),
 			lastDayOfTheMonth: this.getLastDayOfTheMonth(),
-			needsVerification: config( 'email_verification_gate' ) && userUtils.needsVerificationForSite( this.props.site ),
+			needsVerification: config( 'email_verification_gate' ) && this.props.userUtils.needsVerificationForSite( this.props.site ),
 		};
 	},
 
+	componentWillReceiveProps: function( nextProps ) {
+		this.setState( {
```

Comment:
https://facebook.github.io/react/docs/component-api.html

> `setState()` will always trigger a re-render unless conditional rendering logic is implemented in `shouldComponentUpdate()`. If mutable objects are being used and the logic cannot be implemented in `shouldComponentUpdate()`, calling `setState()` only when the new state differs from the previous state will avoid unnecessary re-renders.

So yeah I was wrong, it always triggers a re-render. Might make sense to add that check, then. (Thought I don't think this will be called frequently)